lib/ansible/modules/database/misc/sqitch.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2018, Dawid Wolski <dawidtomaszwolski@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+module: sqitch
+short_description: Manage databases using sqitch
+description:
+  - Manage PostgreSQL, SQLite, Oracle, MySQL, Firebird and Vertica database changes using sqitch
+version_added: "2.8"
+author: "Dawid Wolski, @merito"
+requirements:
+  - sqitch
+notes:
+  - Requires sqitch installed on the remote host. There are many ways you can install it, all are described on
+    sqitch.org
+options:
+  command:
+    description:
+      - A sqitch command, tells sqitch what kind of action it should take
+    required: true
+    default: deploy
+    choices: ["deploy", "rebase", "revert", "status", "verify"]
+  plan_file:
+    description:
+      - Path to deployment plan file
+    required: no
+  target:
+    description:
+      - Database to which to connect
+    required: no
+  engine:
+    description:
+      - Database engine.
+    required: no
+  cwd:
```

Comment:
This should probably be named `config_dir` rather than `cwd`.

Reply:
I think it is more like the working_directory https://github.com/ansible/ansible/blob/773c0982b0d0f1c7a045a0a92ca0c0485affc814/test/runner/lib/util.py#L309

Reply:
That works too. It's better than just `cwd`.